sdk/batch/Azure.ResourceManager.Batch/tests/Helpers/ResourceDataHelper.cs

```diff
@@ -60,7 +60,7 @@ public static BatchAccountCreateOrUpdateContent GetBatchAccountData()
         {
             var data = new BatchAccountCreateOrUpdateContent(AzureLocation.WestUS)
             {
-                AutoStorage = new BatchAccountAutoStorageBaseConfiguration(new ResourceIdentifier("/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/AutoRestResources2/providers/Microsoft.Storage/storageAccounts/20220725datafactory"))
+                AutoStorage = new BatchAccountAutoStorageBaseConfiguration(new ResourceIdentifier("/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/resourceGroups/AutoRestResources2/providers/Microsoft.Storage/storageAccounts/batchaccounttest"))
```

Comment:
Any reason we have to hardcode a storage account id here instead of creating one in the corresponding test case?

Reply:
fixed